<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCustomPlot plot examples</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0">
       <widget class="QSlider" name="timeSlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLCDNumber" name="levelDisplay">
        <property name="digitCount">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDial" name="levelSignificand">
        <property name="minimum">
         <number>-99</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="notchTarget">
         <double>3.700000000000000</double>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDial" name="widthSignificand">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSlider" name="widthMagnitude">
        <property name="minimum">
         <number>-9</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="8">
       <widget class="QCustomPlot" name="customPlot" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="levelMagnitude">
        <property name="minimum">
         <number>-9</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QCheckBox" name="autoScalling">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLCDNumber" name="widthDisplay">
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
